packages/platform-ios/src/config/index.js

```diff
@@ -47,13 +47,19 @@ export function projectConfig(
   }
 
   const projectPath = path.join(folder, project);
+  const sourceDir = path.dirname(projectPath);
 
   return {
-    sourceDir: path.dirname(projectPath),
+    sourceDir,
     folder,
     pbxprojPath: path.join(projectPath, 'project.pbxproj'),
     podfile: findPodfilePath(projectPath),
-    podspec: userConfig.podspec || findPodspecName(folder),
+    podspecPath:
+      userConfig.podspecPath ||
+      // podspecs are usually placed in the root dir of the library or in the
+      // iOS project path
+      findPodspec(folder) ||
```

Comment:
I wonder whether the order is correct. `sourceDir` is within `folder`, so theoretically speaking, `findPodspec(folder)` will always return the same results as `findPodspec(sourceDir)` and never return `null`.

Reply:
I thought the same but that's not how it works, the glob for podspec is `glob('*.podspec', {cwd})` which only traverses the cwd and goes no further. I concluded this as desired behavior for performance reasons (plus it happens that there are node_modules inside that cwd, and unrelated podspecs as well)

Reply:
Okay, so it only looks at the root level and doesn't go any deeper. Smart.